models/_model.js

```diff
@@ -418,24 +428,30 @@ class Model {
           gravityCLIReporter.addItemsInJson('New Record Account', self.record , `NEW ${self.model} RECORD`);
 
           let encryptedRecord;
+          console.log(1)
```

Comment:
Avoid using console logs and debugging info

Reply:
ok